Documentation/getting-started.md

```diff
@@ -0,0 +1,193 @@
+  - networking.k8s.io
+  resources:
+  - ingresses
+  verbs: ["get", "list", "watch"]
+- nonResourceURLs: ["/metrics"]
+  verbs: ["get"]
+```
+
+Now, create a ClusterRoleBinding to bind the ClusterRole to the Prometheus ServiceAccount.
+
+```yaml mdox-exec="cat example/rbac/prometheus/prometheus-cluster-role-binding.yaml"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: prometheus
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: prometheus
+subjects:
+- kind: ServiceAccount
+  name: prometheus
+  namespace: default
+```
+
+Apply all these manifests to create the necessary RBAC resources. Now you are all set to deploy a Prometheus instance. Here is an example of a basic Prometheus instance manifest.
+
+```yaml mdox-exec="cat example/user-guides/getting-started/prometheus.yaml"
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: prometheus
+spec:
+  serviceAccountName: prometheus
+  resources:
+    requests:
+      memory: 400Mi
+```
+
+To verify that the instance is up and running, run:
+
+```bash
+kubectl get -n default prometheus prometheus -w
+```
+
+For more information, see the [Prometheus Operator RBAC guide]({{< ref "rbac" >}}).
+
+## Deploying Alertmanager
+
+Let us take a simple example that creates 3 replicas of Alertmanager.
+
+```yaml mdox-exec="cat example/user-guides/alerting/alertmanager-example.yaml"
+apiVersion: monitoring.coreos.com/v1
+kind: Alertmanager
+metadata:
+  name: example
+spec:
+  replicas: 3
+```
+
+Wait for all Alertmanager pods to be ready:
+
+```bash
+kubectl get pods -l alertmanager=example -w
+```
+
+However, Alertmanager as it is now is of no use to us. To properly use Alertmanager, it is important to understand the relationship between Prometheus and Alertmanager. Alertmanager is used to:
+
+* Deduplicate alerts received from Prometheus.
+* Silence alerts.
+* Route and send grouped notifications to various integrations (PagerDuty, OpsGenie, mail, chat, …).
+
+So, to put Alertmanager instances to use, you would need to integrate it with Prometheus.
+
+## Integrating Alertmanager With Prometheus
+
+### Exposing the Alertmanager service
+
+To access the Alertmanager interface, you have to expose the service to the outside. For
+simplicity, we use a `NodePort` Service.
+
+```yaml mdox-exec="cat example/user-guides/alerting/alertmanager-example-service.yaml"
+apiVersion: v1
+kind: Service
+metadata:
+  name: alertmanager-example
+spec:
+  type: NodePort
+  ports:
+  - name: web
+    nodePort: 30903
+    port: 9093
+    protocol: TCP
+    targetPort: web
+  selector:
+    alertmanager: example
+```
+
+Once the Service is created, the Alertmanager web server is available under the
+node's IP address on port `30903`.
+
+> Note: Exposing the Alertmanager web server this way may not be an applicable solution. Read more about the possible options in the [Ingress guide](user-guides/exposing-prometheus-and-alertmanager.md).
```

Comment:
The ingress guide doesn't exist yet or was moved somewhere else, right? I'm getting 404 when accessing from the [preview](https://deploy-preview-104--sleepy-hopper-0fdb6b.netlify.app/docs/platform/getting-started/#:~:text=Note%3A%20Exposing%20the%20Alertmanager%20web%20server%20this%20way%20may%20not%20be%20an%20applicable%20solution.%20Read%20more%20about%20the%20possible%20options%20in%20the%20Ingress%20guide.)

Reply:
It will work as I add [this](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/exposing-prometheus-and-alertmanager.md) page in the next PR.